Functions/InModuleScope.Tests.ps1

```diff
@@ -40,6 +40,15 @@ Describe "Executing test code inside a module" {
         }
     }
 
+    It "Can use ScriptBlock inside the module scope" {
+        $ScriptBlockOne = { Write-Output "I am a ScriptBlockOne" }
```

Comment:
I don't see any need to mix the two tests cases together. Could you split them to `Can execute bound ScriptBlock inside the module scope` ({} makes scriptblock bound to the current session) and `Can execute unbound ScriptBlock in module scope` please? :)